ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml

```diff
@@ -24,6 +24,7 @@ Item {
 
     Component.onCompleted: {
         root.password = root.startupStore.getPassword()
+        forcePswInputFocus()
```

Comment:
@faust4exe since I don't see any need for this function to be exposed, can we move it to the `id: d` object?
Also let's rename it to `forcePasswordInputFocus()` (no need to have `forcePswInputFocus ` or `forceNewPswInputFocus ` since that password is a single one for the scope).

Please do the same for in both places.

Reply:
I am new to codebase - but yes - looks like it can be done.

Reply:
I am new to codebase - but yes - looks like it can be done.